rust/crd/src/lib.rs

```diff
@@ -432,9 +432,14 @@ pub struct TrinoConfig {
     /// Time period Pods have to gracefully shut down, e.g. `30m`, `1h` or `2d`. Consult the operator documentation for details.
     #[fragment_attrs(serde(default))]
     pub graceful_shutdown_timeout: Option<Duration>,
```

Comment:
nit: According to style guide spread them out
```suggestion
    pub graceful_shutdown_timeout: Option<Duration>,
 
```